terraform/api.tf

```diff
@@ -301,11 +306,11 @@ resource "aws_cloudfront_distribution" "univaf_api_ecs" {
   )
   enabled     = true
   price_class = "PriceClass_100" # North America
-  aliases = compact([
+  aliases = [
     var.domain_name,
-    length(aws_route53_record.api_www_domain_record) > 0 ? aws_route53_record.api_www_domain_record[0].fqdn : "",
-    length(aws_route53_record.api_ecs_domain_record) > 0 ? aws_route53_record.api_ecs_domain_record[0].fqdn : ""
-  ])
+    "www.${var.domain_name}",
+    "ecs.${var.domain_name}"
+  ]
```

Comment:
Now I realize why we didn’t use references before: they make an dependency loop in this situation.